string-manipulation-1/src/is-vowel.js

```diff
@@ -0,0 +1,18 @@
+/* exported isVowel */
+function isVowel(char) {
+  switch (char) {
+    case 'a':
+    case 'e':
+    case 'i':
+    case 'o':
+    case 'u':
+    case 'A':
+    case 'E':
+    case 'I':
+    case 'O':
+    case 'U':
+      return true;
+    default:
+      return false;
+  }
+}
```

Comment:
Totally works this way! You can also convert the character to lowercase before you check, so that way you don't need to account for both uppercase and lowercase. Here's our snippet
```
function isVowel(character) {
  switch (character.toLowerCase()) {
    case 'a':
    case 'e':
    case 'i':
    case 'o':
    case 'u':
      return true;
    default:
      return false;
  }
}
```